scripts/config.py

```diff
@@ -183,6 +183,8 @@ def include_in_full(name):
             'MBEDTLS_REMOVE_ARC4_CIPHERSUITES',
             'MBEDTLS_RSA_NO_CRT',
             'MBEDTLS_SSL_HW_RECORD_ACCEL',
+            'MBEDTLS_SSL_PROTO_SSL3',
```

Comment:
That's not a good idea: it broke `basic-build-test.sh` which runs `compat.sh -m ssl3` in `config.py full` (and we didn't notice immediately but [that's another story](https://github.com/ARMmbed/mbedtls/issues/3178)). `basic-build-test.sh` enables deprecated options for good reason: they're still part of our test coverage.